remote-api/src/main/java/org/alfresco/rest/api/model/rules/Rule.java

```diff
@@ -60,7 +61,7 @@
      * @param ruleModel - {@link org.alfresco.service.cmr.rule.Rule} service POJO
      * @return {@link Rule} REST model
      */
-    public static Rule from(final org.alfresco.service.cmr.rule.Rule ruleModel)
+    public static Rule from(final org.alfresco.service.cmr.rule.Rule ruleModel, final NamespaceService namespaceService)
```

Comment:
This namespace service is passed through 2 methods down to it's actual use which seems not to clean.
Thus, I think we should introduce a mapping "layer" for mapping from/to service and API models instead of mapping inside the model classes.